### DISCUSSION TITLE
2021-05-13 Dev Platform

### DISCUSSION THREAD

**Discussion by mikedotexe** [near/community#58] [Archived]
Updates related to Dev Platform team: https://github.com/near/dev-platform
Rust-related board: https://github.com/orgs/near/projects/10
Tooling board: https://github.com/orgs/near/projects/13

## Done
**Tooling**
- Tons of Dependabot updates led by Josh
- Updates to docs due to removed commands from NEAR CLI by Josh
- Remove EVM commands from NEAR CLI by Josh
- Ryan reworked testing in NEAR CLI

**Rust SDK**
- Multiple PRs from Austin as he onboards, probably merged close to 10.

**Misc**
- New major version of create-near-app from Serhii
- Eugene made changes to Aurora Engine during audit
- Eugene pushed commits to a partner's contracts during audit

## In Progress
- Need to publish new major version of NEAR CLI (Serhii)
- Awesome progress on the `near-api-js` cookbook from Josh https://github.com/near-examples/cookbook
- Onboarding: Austin and Josh Q.
- Eugene working through security audits, ramping up Austin.
- Discussions between Kladov, Eugene, and Austin regarding improvements to `near-sdk` crate that will be discussed Monday.
- Serhii working on Ledger issue with NEAR CLI brought up by Alex Shevchenko
- Rust onboarding through a presentation Mike will give Friday (tomorrow at the time of this writing)

**Comment by maxzaver** [comment]
@volovyk-s where can one find changes that were incorporate into `create-near-app v2.0.0`? There is no CHANGELOG.md file and the release does not have a description on Github.

**Reply by volovyks** [reply]
There is a short description: https://github.com/near/create-near-app/releases/tag/v2.0.0
I have changed the major version because `Rust auto-setup added` is a breaking change, not because we had lots of new features or changes. In case the user does not have `rustup` and will try to create a project with a contract in `Rust` he will be offered to install `Rust` automatically which will require some additional input.
